supplier_app/views.py

```diff
@@ -410,7 +410,8 @@ def get_context_data(self, **kwargs):
         
         form.fields['afip_registration_file'].initial = taxpayer.afip_registration_file
         form.fields['witholding_taxes_file'].initial = taxpayer.witholding_taxes_file
-        form.fields['eb_entities'].initial = taxpayer.eb_entities
+        # Todo: migrate relation to OneToOne
```

Comment:
It would be nice to put the ticket id in this comment

Reply:
We don't have a ticket yet for that issue. We are focusses on response the customer demands